packages/core/admin/server/controllers/authentication.js

```diff
@@ -149,6 +167,24 @@ module.exports = {
     ctx.status = 204;
   },
 
+  async multiFactorAuthentication(ctx) {
+    const input = ctx.request.body;
+    await validateMultiFactorAuthenticationInput(input);
+    if (input.code !== ctx.session.verificationCode) {
+      // Throw forbidden error if verification code is incorrect
+      ctx.status = 403;
+    } else {
+      // Set token if valid
+      ctx.status = 200;
```

Comment:
Not needed here, as long as you are returning something you don't need to manually set the response code